src/mgr/ActivePyModules.h

```diff
@@ -193,6 +194,27 @@ class ActivePyModules
     return modules.at(name)->finisher;
   }
 
+  void inc_mod_finisher_cnt(const std::string &name) {
```

Comment:
Also, these will not be thread safe. This should be a `std::atomic_uint` in `ActivePyModule` rather than maintained as a map.

Reply:
Applied this suggestion. PTAL